.pipelines/templates/.builder-release-template-windows.yaml

```diff
@@ -122,10 +123,10 @@ steps:
       docker run --rm \
       -v ${PWD}:/go/src/github.com/Azure/AgentBaker \
       -w /go/src/github.com/Azure/AgentBaker \
-      -e CLIENT_ID=${CLIENT_ID} \
-      -e CLIENT_SECRET="$(CLIENT_SECRET)" \
-      -e TENANT_ID=${TENANT_ID} \
-      -e SUBSCRIPTION_ID="${SUBSCRIPTION_ID}" \
+      -e CLIENT_ID=${BUILD_CLIENT_ID} \
```

Comment:
Please use CLIENT_ID but not BUILD_CLIENT_ID since it is used to logon the VHD destination

Reply:
Replied you in a separate email. Just for the record (for the future audience), this is a part where we've been changing back and forth for some time. Needed some concrete and well-organized clarifications.